cluster/sync.sh

```diff
@@ -40,7 +40,7 @@ REGISTRY=$push_registry make docker-build
 REGISTRY=$push_registry make docker-push
 REGISTRY=$manifest_registry make manifests
 
-./cluster/kubectl.sh delete --ignore-not-found -f $bridge_marker_manifest
+sed 's/quay.io\/kubevirt/registry:5000/g' examples/bridge-marker.yml | ./cluster/kubectl.sh delete --ignore-not-found -f -
```

Comment:
I don't think the sed is needed here, is it? I'd guess it only cares about name, namespace and type.

Reply:
Also, why is this not using the bridge_marker_manifest variable?

Reply:
> Also, why is this not using the bridge_marker_manifest variable?

No reason. Done.

Reply:
> I don't think the sed is needed here, is it? I'd guess it only cares about name, namespace and type.

Removed.